src/etheroll/roll.kv

```diff
@@ -17,15 +17,15 @@
         size_hint_y: None
         MDTextField:
             id: bet_size_input_id
-            text: "0.1"
+            text: "0.01"
             font_size: dp(20)
             input_filter: 'float'
             width: dp(40)
             size_hint_x: None
         MDSlider:
             id: bet_size_slider_id
             range: 0, 10
-            step: 0.1
+            step: 0.01
```

Comment:
I was wondering if we should not leave the steps being `0.1`, because I feel like it will bug the majority to have not so rounded numbers. But still allow to edit the field by and to put more decimal (by making `round_digits = 2` like you did).
Or we could make the steps `0.05` so users see we have 2 decimal place, but still the slider is not as cumbersome as with steps of `0.05`. This is probably the best of both worlds. And if users really want to have a number not being a multiple of `0.05` they can still edit the value by tapping on the field and using the keyboard.